cfgov/v1/wagtail_hooks.py

```diff
@@ -16,15 +17,11 @@ def share_the_page(request, page):
     is_sharing = bool(request.POST.get('action-share')) and parent_page_perms.can_publish()
 
     if is_sharing or is_publishing:
-        if isinstance(page, CFGOVPage):
-            page.shared = True
+        page.shared = True
     else:
         page.shared = False
 
     page.save()
-    revision = page.save_revision()
-    if is_publishing:
-        revision.publish()
```

Comment:
The default Page model save. this function is called after that save completes from the hook `after_create_page` && `after_edit_page`